layer/index.js

```diff
@@ -0,0 +1,7 @@
+module.export = {
```

Comment:
Hi, I was trying to use to make some tests. I had some problems fixed `module.exports` instead of `module.export`. 
Does it work for you?